docs/tutorial-2/single-node.rst

```diff
@@ -0,0 +1,122 @@
+===========
+Single-node
+===========
+
+This tutorial shows how to install MicroCeph on a single machine.
+
+Ensure storage requirements
+---------------------------
+
+Three OSDs will be required to form a minimal Ceph cluster. This means that
+three individual disks are required to be available on the host machine.
+
+.. note::
+
+   Upstream Ceph development is underway to allow for loopback device support.
+   This will filter down to MicroCeph which will allow for easier
+   proof-of-concept and developer deployments.
+
+The disk subsystem can be summarised via the :command:`lsblk` command. In this
+tutorial, it looks like this (loopback devices have been suppressed in the
+output for purposes of brevity and clarity:
+
+.. code-block:: none
+
+   lsblk | grep -v loop
+
+   NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+   sda           8:0    0 931.5G  0 disk
+   sdb           8:16   0 931.5G  0 disk
+   sdc           8:32   0 931.5G  0 disk
+   sdd           8:48   0 931.5G  0 disk
+   nvme0n1     259:0    0 372.6G  0 disk
+   ├─nvme0n1p1 259:1    0   512M  0 part /boot/efi
+   └─nvme0n1p2 259:2    0 372.1G  0 part /
+
+There are four disks available, here we will use disks ``/dev/sda``,
+``/dev/sdb``, and ``/dev/sdc``.
```

Comment:
Not sure if it goes without saying, but might want to add a caution that we're going to wipe the data? Either here or below when actually adding the disks